boards/arm/stm32h747i_disco/Kconfig.defconfig

```diff
@@ -38,6 +38,27 @@ config CLOCK_STM32_D3PPRE
 config STM32H7_DUAL_CORE
 	default y
 
+config DISK_ACCESS_STM32_SDMMC
+	default n
+
+config DISK_ACCESS_STM32_CLOCK_SOURCE_PLL2_R
```

Comment:
Afaik choices are usually selected like this:
```
choice DISK_ACCESS_STM32_CLOCK_SOURCE
	default DISK_ACCESS_STM32_CLOCK_SOURCE_PLL2_R
endchoice
```

Example:
https://github.com/zephyrproject-rtos/zephyr/blob/master/boards/shields/buydisplay_2_8_tft_touch_arduino/Kconfig.defconfig#L48

Reply:
Will change it asap

Reply:
If I remove this setting as requested I can't get SDMMC driver to compile... Do I need to add some conditions based on DISK_ACCESS manually?

Reply:
OK, got it to work with changed rule in Kconfig, like this:
```
if DISK_ACCESS

config DISK_ACCESS_STM32_SDMMC
	default y

endif # DISK_ACCESS
```

Reply:
OK, changed the choice as recommended